NOTICE

```diff
@@ -1,10 +1,19 @@
+This component contains software that is Copyright (c) 2020 Metrological
+The component is licensed to you under the Apache License, Version 2.0 (the "License").
+You may not use the component except in compliance with the License.
+
+The component may include material which is licensed under other licenses / copyrights as
+listed below. Your use of this material within the component is also subject to the terms and
+conditions of these licenses. The LICENSE file contains the text of all the licenses which apply
+within this component.
+
 Copyright (C) 2016 TATA ELXSI
-Copyright (C) 2016 Metrological
+
```

Comment:
Please keep this Metrological copyright.

Reply:
Done